Dockerfile

```diff
@@ -1,28 +1,32 @@
 # Client
-FROM public.ecr.aws/bitnami/node:14.17.3 AS client
+FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS client
 
 # Build client
-WORKDIR /client
+# RUN apk add --no-cache git python g++ make
+USER root
+WORKDIR /opt/app-root/src/app/client
 COPY client/package*.json ./
 RUN npm set progress=false && npm ci --no-cache
 COPY client/. .
 RUN INLINE_RUNTIME_CHUNK=false npm run build
-
+USER 1001
 # Server
-FROM public.ecr.aws/bitnami/node:14.17.3 AS server
+FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS server
 
 # Static env vars
 ARG VERSION
 ENV VERSION $VERSION
 ENV NODE_ENV production
 
 # Configure server
-COPY --from=client /client/build /client/build/.
-WORKDIR /server
+# RUN apk add --no-cache git
+USER root
```

Comment:
How are we allowing root for build? Does this apply only for the `tools` workspace? 

Can you point to the relevant config or add more details on how to do it as comments on the code? :)

Reply:
1. This is container specific user (by default Redhat containers are non-root container)
2. Are you talking about commented code?  we are not using those applications  and there is no support for apk package manager in Redhat images (**yum**)

Reply:
I meant, we can add the point 1 with more details as a comment for future readers on why we have enabled root permissions here.

Reply:
I would leave root disabled, and only enable it for specific commands where needed

It's disabled for a reason, right? To protect us from breaking things 😓

Reply:
By default all RHEL images run as non-root user. Here we build app as root but in the end we switch to non-root user.

Reply:
And regular alpine/bitmani or ubuntu based images (like current one) run container as `root`. Please check this [blog](https://engineering.bitnami.com/articles/running-non-root-containers-on-openshift.html)

Reply:
I think it's a bad practice 🤷‍♀️ but I also think the risk is low, so I'm fine with this if no one agrees with me!

Reply:
I'm on-board with upholding best practices around permissions, it's a good habit and I can't see any downsides.